java/src/org/openqa/selenium/net/UrlChecker.java

```diff
@@ -130,9 +130,7 @@ public void waitUntilUnavailable(long timeout, TimeUnit unit, final URL url)
       throw new TimeoutException(String.format(
           "Timed out waiting for %s to become unavailable after %d ms",
           url, MILLISECONDS.convert(System.nanoTime() - start, NANOSECONDS)), e);
-    } catch (RuntimeException e) {
-      throw e;
-    } catch (Exception e) {
+    } catch (InterruptedException | ExecutionException e) {
```

Comment:
If we are handling `InterruptedExcption`, shouldn't be the thread interrupted here?

Reply:
@diemol  Good point. Probably we should interrupt the current thread, yes. 
But... the thread was not interrupted before my change, and I just did a refactoring, so the code behaves exactly the same after my changes. :)